src/get_estimation.py

```diff
@@ -2,32 +2,27 @@
 # -*- coding:utf-8 -*-
 
 
+import warnings
 import time
-from .benchmark_mediation import *
-from rpy2.rinterface_lib.embedded import RRuntimeError
 import sys
+from rpy2.rinterface_lib.embedded import RRuntimeError
 import pandas as pd
 import numpy as np
+from .benchmark_mediation import *
 
 
 def get_estimation(x, t, m, y, estimator, config):
```

Comment:
If the function is meant to be public, please add docstrings.

Reply:
Understood.

Reply:
Docstrings added, cf. c962edb